src/components/ItaliaTheme/Blocks/TextCard/CardWithImage/Sidebar.jsx

```diff
@@ -105,6 +110,14 @@ const Sidebar = ({ data, block, onChangeBlock }) => {
               value={data.sizeImage || 's'}
             />
           </FormFieldWrapper>
+          <TextWidget
+            id="AltText"
```

Comment:
```suggestion
            id="altText"
```

Per coerenza lo mettiamo uguale alla key che hai usato in `data`

Reply:
Fatto!